### PR TITLE
Feature/properly handle link solid shapes

### PR DESCRIPTION
**Pull request by fjandrad**
Added templates to correctly handle the modelSolidShapes type of variable obtained with the method 
model.visualSolidShapes.linkSolidShapes
 
The reference issue is https://github.com/robotology/idyntree/issues/656

### CODE REVIEW

File: bindings/iDynTree.i [hunk 1]
```diff
@@ -302,6 +302,11 @@ TEMPLATE_WRAP_MOTION_FORCE(ForceVector3, WRAP_FORCE, SET_NAME_FOR_WRAPPER,,)
 
 %include "joints.i"
 
+%template(SolidshapesVector) std::vector<iDynTree::SolidShape*>;
+//%template(linksSolidshapesVector) std::vector<SolidshapesVector>;
```
Comment (traversaro) [inline bindings/iDynTree.i]:
```suggestion
```
If this is not useful I think we can simply delete it, no?

**Comment by traversaro** [comment]
The  change seems fine, can you update the  CHANGELOG ( https://github.com/robotology/idyntree/blob/devel/CHANGELOG.md )? Thanks!

**Comment by fjandrad** [comment]
@traversaro  can I merge?

**Comment by traversaro** [comment]
Hi @fjandrad , on iDynTree I tipically handle merge personally being the maintainer. For the future, please reply to the PR when you address a review comment, otherwise the reviewer does not get any notification if you just push to the branch without any comment.

**Comment by fjandrad** [comment]
> please reply to the PR when you address a review comment, otherwise the reviewer does not get any notification if you just push to the branch without any comment.

Will do.

**Pull request merged**